sites/org/components/posts/page-wrapper.mjs

```diff
@@ -0,0 +1,74 @@
+import { PageLink } from 'shared/components/page-link.mjs'
```

Comment:
It should be trivial to add `'newsletter'` to the array of `postTypes ` and then we have newsletter pages too. I just didn't implement it because newsletters aren't accounted for in the nav so I figured we didn't serve them this way.

I do understand that `postType` is not as easy to reason about as `blog` or `showcase`. I'm guessing when you say "shared components", though, you don't mean to let me import the entire page component and also the server-side functions?